openssl.pri

```diff
@@ -2,7 +2,8 @@
     CONFIG(release, debug|release): SSL_PATH = $$PWD
                               else: SSL_PATH = $$PWD/no-asm
 
-    if (versionAtLeast(QT_VERSION, 6.5.0)) {
+    if (versionAtLeast(QT_VERSION, 6.5.0) | if(versionAtLeast(QT_VERSION, 5.15.8) : versionAtMost(QT_VERSION, 6.0.0))) {
+        message(Using OpenSSL v3)
```

Comment:
these prints are done only for qmake, either add to cmake also or remove them to keep things more consistent